tests/e2e/support/commands.js

```diff
@@ -1168,7 +1168,7 @@ Cypress.Commands.add('loadGameFixture', (pNum, fixture) => {
       .window()
       .its('cuttle.gameStore.id')
       .then(async (gameId) => {
-        await cy.makeSocketRequest(`game/${gameId}`, 'loadFixtureGameState', fixture);
+        await cy.makeSocketRequest(`game/${gameId}`, 'loadFixtureGameState', { fixture });
```

Comment:
Is this still necessary? It looks like it was previously used when we were sending a separate chunk of data along with the fixture but now it looks extraneous